include/soralog/macro.hpp

```diff
@@ -24,8 +24,9 @@
 
 namespace soralog::macro {
   template <typename... Args>
-  void proxy(const std::shared_ptr<soralog::Logger> &log, soralog::Level level,
-             std::string_view fmt, Args &&... args) {
+  inline void proxy(const std::shared_ptr<soralog::Logger> &log,
+                    soralog::Level level, std::string_view fmt,
+                    Args &&... args) {
     if (log->level() >= level) {
       log->log(level, fmt, std::move(args)()...);
     }
```

Comment:
Comment for the whole file, not this line: I think that WRAP is quite a common name and it may collide with other symbols in a project soralog is used at, so while we don't use module, it might be better to add SORALOG prefix to WRAP macto.

Reply:
Done